libshpool/src/daemon/server.rs

```diff
@@ -303,8 +310,36 @@ impl Server {
                 error!("error writing reply status: {:?}", e);
             }
 
+            // If in pager motd mode, launch the pager and block until it is
+            // done, picking up any tty size change that happened while the
+            // user was examining the motd.
+            let motd_mode = self.config.motd.clone().unwrap_or_default();
+            let init_tty_size = if matches!(motd_mode, MotdDisplayMode::Pager { .. }) {
+                match self.motd_shower.display_in_pager(
```

Comment:
daily_messenger, as was done for the type in the other PR?

Reply:
done